CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Why such a recent CMake version? This is just a one year old CMake release, won't be available on lots of distro versions.

Side note: Would always add a comment why a certain CMake version was set as minimum required (i.e. `# b/c of feature XZY`)

Reply:
Thanks, will go with 3.13 as discussed.